registry/exceptions.py

```diff
@@ -43,6 +43,13 @@ def __init__(self):
         super().__init__({})
 
 
+class UpdateOrganizationIDException(CustomException):
```

Comment:
This way we end with too many exceptions. Would be good to use generic one like "OperationNotAllowedException".
And want to understand, why this case came in the first place.

Reply:
it is specific to field that user can not update the organization ID. where `OperationNotAllowedException` will be too generic to explain the problem. and if there are multiple update restrictions then we can make it generic `UpdateNotAllowed`